src/sentry/static/sentry/app/views/projectSettings/index.jsx

```diff
@@ -24,10 +21,10 @@ const ProjectSettings = React.createClass({
     this.fetchData();
   },
 
-  routeDidChange(nextPath, nextParams) {
-    var params = this.context.router.getCurrentParams();
-    if (nextParams.projectId != params.projectId ||
-        nextParams.orgId != params.orgId) {
+  componentDidChange(prevProps) {
```

Comment:
should be `componentDidUpdate` ... also, can't call `setState` from `componentDidUpdate`